addons/web/static/src/js/core/service_mixins.js

```diff
@@ -238,43 +238,17 @@ var ServicesMixin = {
      * Displays a notification.
      *
      * @param {Object} options
-     * @param {string} options.title
+     * @param {string} [options.title]
```

Comment:
That's ancillary to this change, but maybe we should check if there's a title or a message? Seems useless to show a notification with neither.

Reply:
It's a programming error but if we can help developers detect the mistake why not.

Reply:
I tried to prevent it but as per test "Display a custom notification" we apparently want to support case where the notification content is fully custom and so none of those should be set. We could probably design it better around this, but as all I really want to do here is to make it safe by default I'll return to no check here.